server/src/services/userService.ts

```diff
@@ -113,6 +113,20 @@ class UserService {
       throw error;
     }
   }
+
+  static async patchImagesUploadedStatus(id, status) {
```

Comment:
Weird name. You also called it updateImageStatus in Controller. Be consistent with the naming.